src/SlnParser.Tests/IntegrationTests.cs

```diff
@@ -327,6 +327,78 @@ public void Should_Be_Able_To_Parse_TestSln_Solution_Correctly()
                 .Contain(file => file.Name == "testNested1.txt");
         }
 
+        [Fact]
+        public void Parse_WithProjectWithoutPlatform_IsParsedCorrectly()
+        {
+            string solutionFilePath = GetTempFileName(".sln");
+
+            try
+            {
+                File.WriteAllText(solutionFilePath, @"﻿
+Microsoft Visual Studio Solution File, Format Version 10.00
+# Visual Studio 2008
+Project(""{D183A3D8-5FD8-494B-B014-37F57B35E655}"") = ""Test"", ""Test.dtproj"", ""{D5BDBC46-CEAF-4C92-8335-31450B76914F}""
+EndProject
+Global
+        GlobalSection(SolutionConfigurationPlatforms) = preSolution
+                SolutionConfigurationName|SolutionPlatformName = SolutionConfigurationName|SolutionPlatformName
+        EndGlobalSection
+        GlobalSection(ProjectConfigurationPlatforms) = postSolution
+                {D5BDBC46-CEAF-4C92-8335-31450B76914F}.SolutionConfigurationName|SolutionPlatformName.ActiveCfg = ProjectConfigurationName
+                {D5BDBC46-CEAF-4C92-8335-31450B76914F}.SolutionConfigurationName|SolutionPlatformName.Build.0 = ProjectConfigurationName
+        EndGlobalSection
+EndGlobal");
```

Comment:
Please create a new SolutionFile inside the `Solutions`-folder, see [here](https://github.com/wgnf/SlnParser/tree/main/src/SlnParser.Tests/Solutions). You just need to set the file as `CopyIfNewer` and then you can use it with `LoadSolution("YourSolution")` (You could actually configure the whole folder that everything should be copied from there with 
```xml
<None Include="Solutions\*.sln">
    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
```

).  
  
You don't have to create any new files and make sure that they're removed that way.

Reply:
Thank you.

Sometimes I find it easier to have the test's input values in the same scope as the test assertions so it's easier to understand instead of having to switch windows/context.

Reply:
You are right, but I also think that working with files in tests, that you have to delete and such, makes the test more complex. Its a matter of weighing out the possible options.